### ISSUE TITLE
🐛 'noIrreguluarWhitespace' missing in 'configuration_schema.json'

### ISSUE THREAD

**Issue by CHE1RON** [biomejs/biome#3877]
### Environment information

```block
CLI:
  Version:                      1.9.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.16.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "pnpm/9.10.0"

Biome Configuration:
  Error:                        Found an unknown key `noIrreguluarWhitespace`.
  Status:                       Loaded with errors
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Workspace:
  Open Documents:               0
```

### What happened?

Title says it all, including `noIrreguluarWhitespace` after v1.9 upgrade throws `Found an unknown key` :shrug:

### Expected result

`noIrreguluarWhitespace` should not throw an error.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by Conaclos** [comment]
There is a typo: `noIrreguluarWhitespace` -> `noIrregularWhitespace`

**Comment by CHE1RON** [comment]
Just copied it from the v1.9 release page, fyi 😉 

Thanks! 🙏

**Comment by nhedger** [comment]
Thanks, it was fixed in [71f967f](https://github.com/biomejs/website/pull/1113)
